pkgsStatic.audiofile failes to compile

**Describe the bug**

`nix-build -A pkgsStatic.audiofile`

```
❯ nb -A pkgsStatic.audiofile
these derivations will be built:
  /nix/store/zswyqwjxifag7qp3mi8wyz9206gn77bk-audiofile-0.3.6-x86_64-unknown-linux-musl.drv
building '/nix/store/zswyqwjxifag7qp3mi8wyz9206gn77bk-audiofile-0.3.6-x86_64-unknown-linux-musl.drv'...
unpacking sources
unpacking source archive /nix/store/5s5rlwc2i61lmqdddvnkj3mmybmjawyp-audiofile-0.3.6.tar.gz
source root is audiofile-0.3.6
setting SOURCE_DATE_EPOCH to timestamp 1362641240 of file audiofile-0.3.6/config.h.in
patching sources
applying patch /nix/store/z3pqhg6lbygziw6gbv0nwmmsm81s6lfi-gcc-6.patch
patching file libaudiofile/modules/SimpleModule.h
applying patch /nix/store/1cl09g0szzjgchiy7dz2xs8w5nqi2004-CVE-2015-7747.patch
patching file libaudiofile/modules/ModuleState.cpp
patching file test/Makefile.am
patching file test/sixteen-stereo-to-eight-mono.c
applying patch /nix/store/z4hnchyvbr4h9zkbjg2ympywqxz5mar8-CVE-2017-6829.patch
patching file libaudiofile/modules/IMA.cpp
applying patch /nix/store/vg3hm494n2hsk62ag1x6safnildsmd6z-CVE-2017-6827+CVE-2017-6828+CVE-2017-6832+CVE-2017-6835+CVE-2017-6837.patch
patching file libaudiofile/WAVE.cpp
applying patch /nix/store/ayf47cah4hazswxp9di8w2l6jpzh9da7-CVE-2017-6839.patch
patching file libaudiofile/modules/BlockCodec.cpp
patching file libaudiofile/modules/MSADPCM.cpp
applying patch /nix/store/kzcbd6a8w99x70llahmbaplgd79hndyc-CVE-2017-6830+CVE-2017-6834+CVE-2017-6836+CVE-2017-6838.patch
patching file sfcommands/sfconvert.c
applying patch /nix/store/mbc8cyiv2iv1lxdnaq5b2n38cckpiz3a-audiofile-fix-multiplyCheckOverflow-signature.patch
patching file libaudiofile/modules/MSADPCM.cpp
patching file sfcommands/sfconvert.c
applying patch /nix/store/yysjxmgn0nqbp78yr8j0hwmxzkbidq9y-CVE-2017-6831.patch
patching file libaudiofile/WAVE.cpp
applying patch /nix/store/y3vg0xg5k0l7psjic7vi76dm5bn72rl1-CVE-2017-6833.patch
patching file libaudiofile/modules/BlockCodec.cpp
updateAutotoolsGnuConfigScriptsPhase
Updating Autotools / GNU config script to a newer upstream version: ./config.sub
Updating Autotools / GNU config script to a newer upstream version: ./config.guess
configuring
fixing libtool script ./ltmain.sh
configure flags: --disable-dependency-tracking --prefix=/nix/store/l4gls1rpv8qc60q0gl5ickcvyg3fls45-audiofile-0.3.6-x86_64-unknown-linux-musl --disable-shared --enable-static --disable-shared --build=x86_64-unknown-linux-gnu --host=x86_64-unknown-linux-musl
checking for a BSD-compatible install... /nix/store/2shqhfsyzz4rnfyysbzgyp5kbfk29750-coreutils-8.32/bin/install -c
checking whether build environment is sane... yes
checking for x86_64-unknown-linux-musl-strip... x86_64-unknown-linux-musl-strip
checking for a thread-safe mkdir -p... /nix/store/2shqhfsyzz4rnfyysbzgyp5kbfk29750-coreutils-8.32/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking for style of include used by make... GNU
checking for x86_64-unknown-linux-musl-gcc... x86_64-unknown-linux-musl-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether x86_64-unknown-linux-musl-gcc accepts -g... yes
checking for x86_64-unknown-linux-musl-gcc option to accept ISO C89... none needed
checking dependency style of x86_64-unknown-linux-musl-gcc... none
checking for x86_64-unknown-linux-musl-gcc option to accept ISO C99... none needed
checking whether we are using the GNU C++ compiler... yes
checking whether x86_64-unknown-linux-musl-g++ accepts -g... yes
checking dependency style of x86_64-unknown-linux-musl-g++... none
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-musl
checking how to print strings... printf
checking for a sed that does not truncate output... /nix/store/i9ka2lyn3kb48fhqp8n3qazqkc6ks5k1-gnused-4.8/bin/sed
checking for grep that handles long lines and -e... /nix/store/8kv1647yhqd13z571h56rgm2ldm567dm-gnugrep-3.5/bin/grep
checking for egrep... /nix/store/8kv1647yhqd13z571h56rgm2ldm567dm-gnugrep-3.5/bin/grep -E
checking for fgrep... /nix/store/8kv1647yhqd13z571h56rgm2ldm567dm-gnugrep-3.5/bin/grep -F
checking for ld used by x86_64-unknown-linux-musl-gcc... x86_64-unknown-linux-musl-ld
checking if the linker (x86_64-unknown-linux-musl-ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... x86_64-unknown-linux-musl-nm
checking the name lister (x86_64-unknown-linux-musl-nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 32768
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-musl format... func_convert_file_noop
checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for x86_64-unknown-linux-musl-ld option to reload object files... -r
checking for x86_64-unknown-linux-musl-objdump... x86_64-unknown-linux-musl-objdump
checking how to recognize dependent libraries... pass_all
checking for x86_64-unknown-linux-musl-dlltool... no
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for x86_64-unknown-linux-musl-ar... x86_64-unknown-linux-musl-ar
checking for archiver @FILE support... @
checking for x86_64-unknown-linux-musl-strip... (cached) x86_64-unknown-linux-musl-strip
checking for x86_64-unknown-linux-musl-ranlib... x86_64-unknown-linux-musl-ranlib
checking command to parse x86_64-unknown-linux-musl-nm output from x86_64-unknown-linux-musl-gcc object... ok
checking for sysroot... no
./configure: line 6910: /usr/bin/file: No such file or directory
checking for x86_64-unknown-linux-musl-mt... no
checking for mt... no
checking if : is a manifest tool... no
checking how to run the C preprocessor... x86_64-unknown-linux-musl-gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if x86_64-unknown-linux-musl-gcc supports -fno-rtti -fno-exceptions... no
checking for x86_64-unknown-linux-musl-gcc option to produce PIC... -fPIC -DPIC
checking if x86_64-unknown-linux-musl-gcc PIC flag -fPIC -DPIC works... yes
checking if x86_64-unknown-linux-musl-gcc static flag -static works... yes
checking if x86_64-unknown-linux-musl-gcc supports -c -o file.o... yes
checking if x86_64-unknown-linux-musl-gcc supports -c -o file.o... (cached) yes
checking whether the x86_64-unknown-linux-musl-gcc linker (x86_64-unknown-linux-musl-ld) supports shared libraries... yes
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... no
checking whether to build static libraries... yes
checking how to run the C++ preprocessor... x86_64-unknown-linux-musl-g++ -E
checking for ld used by x86_64-unknown-linux-musl-g++... x86_64-unknown-linux-musl-ld
checking if the linker (x86_64-unknown-linux-musl-ld) is GNU ld... yes
checking whether the x86_64-unknown-linux-musl-g++ linker (x86_64-unknown-linux-musl-ld) supports shared libraries... yes
checking for x86_64-unknown-linux-musl-g++ option to produce PIC... -fPIC -DPIC
checking if x86_64-unknown-linux-musl-g++ PIC flag -fPIC -DPIC works... yes
checking if x86_64-unknown-linux-musl-g++ static flag -static works... yes
checking if x86_64-unknown-linux-musl-g++ supports -c -o file.o... yes
checking if x86_64-unknown-linux-musl-g++ supports -c -o file.o... (cached) yes
checking whether the x86_64-unknown-linux-musl-g++ linker (x86_64-unknown-linux-musl-ld) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for ANSI C header files... (cached) yes
checking fcntl.h usability... yes
checking fcntl.h presence... yes
checking for fcntl.h... yes
checking for unistd.h... (cached) yes
checking for an ANSI C-conforming const... yes
checking whether byte ordering is bigendian... no
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking for off_t... yes
checking for size_t... yes
checking for platform specific tests to compile... linux
checking for a2x... :
checking for asciidoc... :
configure: WARNING: Could not find a2x.
configure: WARNING: Could not find asciidoc.
checking for x86_64-unknown-linux-musl-pkg-config... no
checking for pkg-config... no
configure: creating ./config.status
config.status: creating audiofile.spec
config.status: creating audiofile.pc
config.status: creating audiofile-uninstalled.pc
config.status: creating sfcommands/Makefile
config.status: creating test/Makefile
config.status: creating gtest/Makefile
config.status: creating examples/Makefile
config.status: creating libaudiofile/Makefile
config.status: creating libaudiofile/alac/Makefile
config.status: creating libaudiofile/modules/Makefile
config.status: creating docs/Makefile
config.status: creating Makefile
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
building
build flags: SHELL=/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash
make  all-recursive
make[1]: Entering directory '/build/audiofile-0.3.6'
Making all in gtest
make[2]: Entering directory '/build/audiofile-0.3.6/gtest'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/build/audiofile-0.3.6/gtest'
Making all in libaudiofile
make[2]: Entering directory '/build/audiofile-0.3.6/libaudiofile'
Making all in alac
make[3]: Entering directory '/build/audiofile-0.3.6/libaudiofile/alac'
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../..    -DNDEBUG -Wall -Wno-multichar  -fvisibility=hidden -g -O2 -c -o ag_dec.lo ag_dec.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -g -O2 -c ag_dec.c -o ag_dec.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../..    -DNDEBUG -Wall -Wno-multichar  -fvisibility=hidden -g -O2 -c -o ag_enc.lo ag_enc.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -g -O2 -c ag_enc.c -o ag_enc.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../..    -DNDEBUG -Wall -Wno-multichar  -fvisibility=hidden -g -O2 -c -o ALACBitUtilities.lo ALACBitUtilities.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -g -O2 -c ALACBitUtilities.c -o ALACBitUtilities.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../..    -DNDEBUG -Wall -Wno-multichar  -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o ALACDecoder.lo ALACDecoder.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c ALACDecoder.cpp -o ALACDecoder.o
ALACDecoder.cpp: In member function 'int32_t ALACDecoder::Decode(BitBuffer*, uint8_t*, uint32_t, uint32_t, uint32_t*)':
ALACDecoder.cpp:185:14: warning: variable 'samples' set but not used [-Wunused-but-set-variable]
  185 |  int16_t *   samples;
      |              ^~~~~~~
ALACDecoder.cpp: In member function 'int32_t ALACDecoder::DataStreamElement(BitBuffer*)':
ALACDecoder.cpp:655:11: warning: variable 'element_instance_tag' set but not used [-Wunused-but-set-variable]
  655 |  uint8_t  element_instance_tag;
      |           ^~~~~~~~~~~~~~~~~~~~
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../..    -DNDEBUG -Wall -Wno-multichar  -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o ALACEncoder.lo ALACEncoder.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c ALACEncoder.cpp -o ALACEncoder.o
ALACEncoder.cpp: In member function 'int32_t ALACEncoder::EncodeStereo(BitBuffer*, void*, uint32_t, uint32_t, uint32_t)':
ALACEncoder.cpp:279:13: warning: variable 'denShift' set but not used [-Wunused-but-set-variable]
  279 |  uint32_t   denShift;
      |             ^~~~~~~~
ALACEncoder.cpp: In member function 'int32_t ALACEncoder::EncodeStereoFast(BitBuffer*, void*, uint32_t, uint32_t, uint32_t)':
ALACEncoder.cpp:558:13: warning: variable 'denShift' set but not used [-Wunused-but-set-variable]
  558 |  uint32_t   denShift;
      |             ^~~~~~~~
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../..    -DNDEBUG -Wall -Wno-multichar  -fvisibility=hidden -g -O2 -c -o dp_dec.lo dp_dec.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -g -O2 -c dp_dec.c -o dp_dec.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../..    -DNDEBUG -Wall -Wno-multichar  -fvisibility=hidden -g -O2 -c -o dp_enc.lo dp_enc.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -g -O2 -c dp_enc.c -o dp_enc.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../..    -DNDEBUG -Wall -Wno-multichar  -fvisibility=hidden -g -O2 -c -o EndianPortable.lo EndianPortable.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -g -O2 -c EndianPortable.c -o EndianPortable.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../..    -DNDEBUG -Wall -Wno-multichar  -fvisibility=hidden -g -O2 -c -o matrix_dec.lo matrix_dec.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -g -O2 -c matrix_dec.c -o matrix_dec.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../..    -DNDEBUG -Wall -Wno-multichar  -fvisibility=hidden -g -O2 -c -o matrix_enc.lo matrix_enc.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I../.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -g -O2 -c matrix_enc.c -o matrix_enc.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=link x86_64-unknown-linux-musl-g++ -DNDEBUG -Wall -Wno-multichar  -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2   -o libalac.la  ag_dec.lo ag_enc.lo ALACBitUtilities.lo ALACDecoder.lo ALACEncoder.lo dp_dec.lo dp_enc.lo EndianPortable.lo matrix_dec.lo matrix_enc.lo
libtool: link: x86_64-unknown-linux-musl-ar cru .libs/libalac.a  ag_dec.o ag_enc.o ALACBitUtilities.o ALACDecoder.o ALACEncoder.o dp_dec.o dp_enc.o EndianPortable.o matrix_dec.o matrix_enc.o
x86_64-unknown-linux-musl-ar: `u' modifier ignored since `D' is the default (see `U')
libtool: link: x86_64-unknown-linux-musl-ranlib .libs/libalac.a
libtool: link: ( cd ".libs" && rm -f "libalac.la" && ln -s "../libalac.la" "libalac.la" )
make[3]: Leaving directory '/build/audiofile-0.3.6/libaudiofile/alac'
Making all in modules
make[3]: Entering directory '/build/audiofile-0.3.6/libaudiofile/modules'
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o ALAC.lo ALAC.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c ALAC.cpp -o ALAC.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o BlockCodec.lo BlockCodec.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c BlockCodec.cpp -o BlockCodec.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o FLAC.lo FLAC.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c FLAC.cpp -o FLAC.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o FileModule.lo FileModule.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c FileModule.cpp -o FileModule.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o G711.lo G711.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c G711.cpp -o G711.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o IMA.lo IMA.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c IMA.cpp -o IMA.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o Module.lo Module.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c Module.cpp -o Module.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o ModuleState.lo ModuleState.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c ModuleState.cpp -o ModuleState.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o MSADPCM.lo MSADPCM.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c MSADPCM.cpp -o MSADPCM.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o PCM.lo PCM.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c PCM.cpp -o PCM.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o RebufferModule.lo RebufferModule.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c RebufferModule.cpp -o RebufferModule.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./..   -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o SimpleModule.lo SimpleModule.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I../.. -I./.. -DNDEBUG -Wall -Wno-multichar -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c SimpleModule.cpp -o SimpleModule.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../../libtool --tag=CXX   --mode=link x86_64-unknown-linux-musl-g++ -DNDEBUG -Wall -Wno-multichar    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2   -o libmodules.la  ALAC.lo BlockCodec.lo FLAC.lo FileModule.lo G711.lo IMA.lo Module.lo ModuleState.lo MSADPCM.lo PCM.lo RebufferModule.lo SimpleModule.lo
libtool: link: x86_64-unknown-linux-musl-ar cru .libs/libmodules.a  ALAC.o BlockCodec.o FLAC.o FileModule.o G711.o IMA.o Module.o ModuleState.o MSADPCM.o PCM.o RebufferModule.o SimpleModule.o
x86_64-unknown-linux-musl-ar: `u' modifier ignored since `D' is the default (see `U')
libtool: link: x86_64-unknown-linux-musl-ranlib .libs/libmodules.a
libtool: link: ( cd ".libs" && rm -f "libmodules.la" && ln -s "../libmodules.la" "libmodules.la" )
make[3]: Leaving directory '/build/audiofile-0.3.6/libaudiofile/modules'
make[3]: Entering directory '/build/audiofile-0.3.6/libaudiofile'
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o AIFF.lo AIFF.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c AIFF.cpp -o AIFF.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o AVR.lo AVR.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c AVR.cpp -o AVR.o
AVR.cpp: In member function 'virtual status AVRFile::writeInit(AFfilesetup)':
AVR.cpp:299:10: warning: 'char* strncpy(char*, const char*, size_t)' specified bound 8 equals destination size [-Wstringop-truncation]
  299 |   strncpy(name, af_basename(m_fileName), 8);
      |   ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o AudioFormat.lo AudioFormat.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c AudioFormat.cpp -o AudioFormat.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o Buffer.lo Buffer.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c Buffer.cpp -o Buffer.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o CAF.lo CAF.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c CAF.cpp -o CAF.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o FLACFile.lo FLACFile.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c FLACFile.cpp -o FLACFile.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o File.lo File.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c File.cpp -o File.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o FileHandle.lo FileHandle.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c FileHandle.cpp -o FileHandle.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o IFF.lo IFF.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c IFF.cpp -o IFF.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o IRCAM.lo IRCAM.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c IRCAM.cpp -o IRCAM.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o Instrument.lo Instrument.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c Instrument.cpp -o Instrument.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o Loop.lo Loop.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c Loop.cpp -o Loop.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o Marker.lo Marker.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c Marker.cpp -o Marker.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o Miscellaneous.lo Miscellaneous.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c Miscellaneous.cpp -o Miscellaneous.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o NIST.lo NIST.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c NIST.cpp -o NIST.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o NeXT.lo NeXT.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c NeXT.cpp -o NeXT.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o PacketTable.lo PacketTable.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c PacketTable.cpp -o PacketTable.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o Raw.lo Raw.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c Raw.cpp -o Raw.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o SampleVision.lo SampleVision.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c SampleVision.cpp -o SampleVision.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o Setup.lo Setup.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c Setup.cpp -o Setup.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o Track.lo Track.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c Track.cpp -o Track.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o UUID.lo UUID.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c UUID.cpp -o UUID.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o VOC.lo VOC.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c VOC.cpp -o VOC.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o WAVE.lo WAVE.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c WAVE.cpp -o WAVE.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o aes.lo aes.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c aes.cpp -o aes.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o af_vfs.lo af_vfs.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c af_vfs.cpp -o af_vfs.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -g -O2 -c -o aupv.lo aupv.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -g -O2 -c aupv.c -o aupv.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o compression.lo compression.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c compression.cpp -o compression.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o data.lo data.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c data.cpp -o data.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o debug.lo debug.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c debug.cpp -o debug.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -g -O2 -c -o error.lo error.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -g -O2 -c error.c -o error.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -g -O2 -c -o extended.lo extended.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -g -O2 -c extended.c -o extended.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o format.lo format.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c format.cpp -o format.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CC   --mode=compile x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -g -O2 -c -o g711.lo g711.c
libtool: compile:  x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -g -O2 -c g711.c -o g711.o
g711.c:59:28: warning: '_a2u' defined but not used [-Wunused-const-variable=]
   59 | static const unsigned char _a2u[128] = {  /* A- to u-law conversions */
      |                            ^~~~
g711.c:41:28: warning: '_u2a' defined but not used [-Wunused-const-variable=]
   41 | static const unsigned char _u2a[128] = {  /* u- to A-law conversions */
      |                            ^~~~
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o openclose.lo openclose.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c openclose.cpp -o openclose.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o pcm.lo pcm.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c pcm.cpp -o pcm.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o query.lo query.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c query.cpp -o query.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o units.lo units.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c units.cpp -o units.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=compile x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o util.lo util.cpp
libtool: compile:  x86_64-unknown-linux-musl-g++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c util.cpp -o util.o
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CXX   --mode=link x86_64-unknown-linux-musl-g++ -DNDEBUG -Wall    -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -version-info 1:0:0 -no-undefined -export-symbols ./audiofile.exports  -o libaudiofile.la -rpath /nix/store/l4gls1rpv8qc60q0gl5ickcvyg3fls45-audiofile-0.3.6-x86_64-unknown-linux-musl/lib AIFF.lo AVR.lo AudioFormat.lo Buffer.lo CAF.lo FLACFile.lo File.lo FileHandle.lo IFF.lo IRCAM.lo Instrument.lo Loop.lo Marker.lo Miscellaneous.lo NIST.lo NeXT.lo PacketTable.lo Raw.lo SampleVision.lo Setup.lo Track.lo UUID.lo VOC.lo WAVE.lo aes.lo af_vfs.lo aupv.lo compression.lo data.lo debug.lo error.lo extended.lo format.lo g711.lo openclose.lo pcm.lo query.lo units.lo util.lo modules/libmodules.la alac/libalac.la   -lm
libtool: link: (cd .libs/libaudiofile.lax/libmodules.a && x86_64-unknown-linux-musl-ar x "/build/audiofile-0.3.6/libaudiofile/modules/.libs/libmodules.a")
libtool: link: (cd .libs/libaudiofile.lax/libalac.a && x86_64-unknown-linux-musl-ar x "/build/audiofile-0.3.6/libaudiofile/alac/.libs/libalac.a")
libtool: link: x86_64-unknown-linux-musl-ar cru .libs/libaudiofile.a  AIFF.o AVR.o AudioFormat.o Buffer.o CAF.o FLACFile.o File.o FileHandle.o IFF.o IRCAM.o Instrument.o Loop.o Marker.o Miscellaneous.o NIST.o NeXT.o PacketTable.o Raw.o SampleVision.o Setup.o Track.o UUID.o VOC.o WAVE.o aes.o af_vfs.o aupv.o compression.o data.o debug.o error.o extended.o format.o g711.o openclose.o pcm.o query.o units.o util.o  .libs/libaudiofile.lax/libmodules.a/ALAC.o .libs/libaudiofile.lax/libmodules.a/BlockCodec.o .libs/libaudiofile.lax/libmodules.a/FLAC.o .libs/libaudiofile.lax/libmodules.a/FileModule.o .libs/libaudiofile.lax/libmodules.a/G711.o .libs/libaudiofile.lax/libmodules.a/IMA.o .libs/libaudiofile.lax/libmodules.a/MSADPCM.o .libs/libaudiofile.lax/libmodules.a/Module.o .libs/libaudiofile.lax/libmodules.a/ModuleState.o .libs/libaudiofile.lax/libmodules.a/PCM.o .libs/libaudiofile.lax/libmodules.a/RebufferModule.o .libs/libaudiofile.lax/libmodules.a/SimpleModule.o  .libs/libaudiofile.lax/libalac.a/ALACBitUtilities.o .libs/libaudiofile.lax/libalac.a/ALACDecoder.o .libs/libaudiofile.lax/libalac.a/ALACEncoder.o .libs/libaudiofile.lax/libalac.a/EndianPortable.o .libs/libaudiofile.lax/libalac.a/ag_dec.o .libs/libaudiofile.lax/libalac.a/ag_enc.o .libs/libaudiofile.lax/libalac.a/dp_dec.o .libs/libaudiofile.lax/libalac.a/dp_enc.o .libs/libaudiofile.lax/libalac.a/matrix_dec.o .libs/libaudiofile.lax/libalac.a/matrix_enc.o
x86_64-unknown-linux-musl-ar: `u' modifier ignored since `D' is the default (see `U')
libtool: link: x86_64-unknown-linux-musl-ranlib .libs/libaudiofile.a
libtool: link: rm -fr .libs/libaudiofile.lax
libtool: link: ( cd ".libs" && rm -f "libaudiofile.la" && ln -s "../libaudiofile.la" "libaudiofile.la" )
make[3]: Leaving directory '/build/audiofile-0.3.6/libaudiofile'
make[2]: Leaving directory '/build/audiofile-0.3.6/libaudiofile'
Making all in sfcommands
make[2]: Entering directory '/build/audiofile-0.3.6/sfcommands'
x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I.. -I../libaudiofile   -Wall  -g -O2 -c sfconvert.c
x86_64-unknown-linux-musl-gcc -DHAVE_CONFIG_H -I. -I.. -I../libaudiofile   -Wall  -g -O2 -c printinfo.c
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CC   --mode=link x86_64-unknown-linux-musl-gcc -Wall  -g -O2   -o sfconvert sfconvert.o printinfo.o ../libaudiofile/libaudiofile.la
libtool: link: x86_64-unknown-linux-musl-gcc -Wall -g -O2 -o sfconvert sfconvert.o printinfo.o  ../libaudiofile/.libs/libaudiofile.a /nix/store/bmsy7ppz0pmcv7xc49cq4x17yvfb0wqn-x86_64-unknown-linux-musl-stage-final-gcc-debug-9.3.0-lib/x86_64-unknown-linux-musl/lib/../lib64/libstdc++.so -Wl,-rpath -Wl,/nix/store/bmsy7ppz0pmcv7xc49cq4x17yvfb0wqn-x86_64-unknown-linux-musl-stage-final-gcc-debug-9.3.0-lib/x86_64-unknown-linux-musl/lib/../lib64 -Wl,-rpath -Wl,/nix/store/bmsy7ppz0pmcv7xc49cq4x17yvfb0wqn-x86_64-unknown-linux-musl-stage-final-gcc-debug-9.3.0-lib/x86_64-unknown-linux-musl/lib/../lib64
/nix/store/gz6ialhn1hv1ripmkbda7ar6gjb2mksy-x86_64-unknown-linux-musl-binutils-2.31.1/bin/x86_64-unknown-linux-musl-ld: attempted static link of dynamic object `/nix/store/bmsy7ppz0pmcv7xc49cq4x17yvfb0wqn-x86_64-unknown-linux-musl-stage-final-gcc-debug-9.3.0-lib/x86_64-unknown-linux-musl/lib/../lib64/libstdc++.so'
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:320: sfconvert] Error 1
make[2]: Leaving directory '/build/audiofile-0.3.6/sfcommands'
make[1]: *** [Makefile:408: all-recursive] Error 1
make[1]: Leaving directory '/build/audiofile-0.3.6'
make: *** [Makefile:312: all] Error 2
builder for '/nix/store/zswyqwjxifag7qp3mi8wyz9206gn77bk-audiofile-0.3.6-x86_64-unknown-linux-musl.drv' failed with exit code 2
error: build of '/nix/store/zswyqwjxifag7qp3mi8wyz9206gn77bk-audiofile-0.3.6-x86_64-unknown-linux-musl.drv' failed
```

**Notify maintainers**

@lovek323

Had a play with this. Looks like at the moment it's trying to link against libstdc++.so, passing `AM_LDFLAGS=-all-static` causes it to try to link against libstdc++.a from the wrong directory 

```
[nix-shell:~/src/test-builds/audiofile-0.3.6]$ make AM_LDFLAGS=-all-static
make  all-recursive
make[1]: Entering directory '/home/j/src/test-builds/audiofile-0.3.6'
Making all in gtest
make[2]: Entering directory '/home/j/src/test-builds/audiofile-0.3.6/gtest'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/j/src/test-builds/audiofile-0.3.6/gtest'
Making all in libaudiofile
make[2]: Entering directory '/home/j/src/test-builds/audiofile-0.3.6/libaudiofile'
Making all in alac
make[3]: Entering directory '/home/j/src/test-builds/audiofile-0.3.6/libaudiofile/alac'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/j/src/test-builds/audiofile-0.3.6/libaudiofile/alac'
Making all in modules
make[3]: Entering directory '/home/j/src/test-builds/audiofile-0.3.6/libaudiofile/modules'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/j/src/test-builds/audiofile-0.3.6/libaudiofile/modules'
make[3]: Entering directory '/home/j/src/test-builds/audiofile-0.3.6/libaudiofile'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/home/j/src/test-builds/audiofile-0.3.6/libaudiofile'
make[2]: Leaving directory '/home/j/src/test-builds/audiofile-0.3.6/libaudiofile'
Making all in sfcommands
make[2]: Entering directory '/home/j/src/test-builds/audiofile-0.3.6/sfcommands'
/nix/store/qdp56fi357fgxxnkjrwx1g67hrk775im-bash-4.4-p23/bin/bash ../libtool --tag=CC   --mode=link x86_64-unknown-linux-musl-gcc -Wall  -g -O2 -all-static  -o sfconvert sfconvert.o printinfo.o ../libaudiofile/libaudiofile.la
libtool: link: x86_64-unknown-linux-musl-gcc -Wall -g -O2 -static -o sfconvert sfconvert.o printinfo.o  ../libaudiofile/.libs/libaudiofile.a /nix/store/bmsy7ppz0pmcv7xc49cq4x17yvfb0wqn-x86_64-unknown-linux-musl-stage-final-gcc-debug-9.3.0-lib/x86_64-unknown-linux-musl/lib/../lib64/libstdc++.a
x86_64-unknown-linux-musl-gcc: error: /nix/store/bmsy7ppz0pmcv7xc49cq4x17yvfb0wqn-x86_64-unknown-linux-musl-stage-final-gcc-debug-9.3.0-lib/x86_64-unknown-linux-musl/lib/../lib64/libstdc++.a: No such file or directory
make[2]: *** [Makefile:320: sfconvert] Error 1
make[2]: Leaving directory '/home/j/src/test-builds/audiofile-0.3.6/sfcommands'
make[1]: *** [Makefile:408: all-recursive] Error 1
make[1]: Leaving directory '/home/j/src/test-builds/audiofile-0.3.6'
make: *** [Makefile:312: all] Error 2
```

--- a/pkgs/development/libraries/audiofile/default.nix
+++ b/pkgs/development/libraries/audiofile/default.nix
@@ -28,6 +28,21 @@ stdenv.mkDerivation rec {
   # fix build with gcc9
   NIX_CFLAGS_LINK = lib.optional (stdenv.system == "i686-linux") "-lgcc";
 
+  # Even when statically linking, libstdc++.la is put in dependency_libs here,
+  # and hence libstdc++.so passed to the linker, just pass -lstdc++ and let the
+  # compiler do what it does best.  (libaudiofile.la is a generated file, so we
+  # have to run `make` that far first).
+  #
+  # Without this, the executables in this package (sfcommands and examples)
+  # fail to build: https://github.com/NixOS/nixpkgs/issues/103215
+  #
+  # There might be a more sensible way to do this with autotools, but I am not
+  # smart enough to discover it.
+  preBuild = lib.optionalString stdenv.targetPlatform.isStatic ''
+    make -C libaudiofile $makeFlags
+    sed -i "s/dependency_libs=.*/dependency_libs=' -lstdc++'/" libaudiofile/libaudiofile.la
+  '';
+
   patches = [
     ./gcc-6.patch
     ./CVE-2015-7747.patch

